### ISSUE TITLE
 In High contrast black/whitemode, the dropdown visible focus indicator is not visible

### ISSUE THREAD

**Issue by benjie-tong** [microsoft/fluentui#28132]
### Environment Information

- **Package version(s)**: Fluent React 8.109.10
- **Browser and OS versions**: Windows Edge

### Describe the issue:

 In High contrast black/whitemode, the dropdown visible focus indicator is not visible

#### Actual behavior:

![image](https://github.com/microsoft/fluentui/assets/49033687/03cdacd9-0e59-47f5-874d-13ab9f45a2d9)

#### Expected behavior:

<img width="168" alt="image" src="https://github.com/microsoft/fluentui/assets/49033687/9e61e007-9562-4cf6-9af8-f98384787357">

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)
